NewSpring/ApollosProject
# [DEV-3771] Fixes Live Banner

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR does 2 things:
1. It updates the live banner so that when it's tapped it now goes to NewSpring's instance of Church Online.
2. It takes advantage of new configuration options so that the live banner will only display when NewSpring is actually live. We actually use a Church Online API endpoint to determine when this is. https://live.newspring.cc/api/v1/events/current

### How do I test this PR?
Open the sim.
If we are currently supposed to be live (see the API endpoint above) then the live banner should show.
If you tap on the live banner it should take you to NewSpring's instance of Church Online.
If you're testing this during a time when we are not live, you can temporarily change the API endpoint to be the one that Apollos Church currently uses (https://apollos.churchonline.org/api/v1/) so that the banner will show in order to test that the link goes where it says it should.

*Note*: This does use version `0.8.7-alpha.1` of `apollos-data-connector-church-online`. Obviously, this is an alpha version that will need to be set to the real `0.8.7` version when it is released.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3771](https://newspring.atlassian.net/browse/DEV-3771)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
